packages/expo-dev-launcher/react-native.config.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this supposed to be JSON?

Reply:
copied over from what brent implemented in expo-random

Reply:
That file might not be right: https://github.com/react-native-community/cli/blob/master/docs/platforms.md#how-does-it-work

Reply:
Seems like this still needs to be fixed